### ISSUE TITLE
Review consistency in components with same functionality (ex: links) 

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/resources-ressources#138]
> Success Criterion [3.2.4 Consistent Identification](https://www.w3.org/TR/WCAG21/#consistent-identification) (Level AA): Components that have the [same functionality](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html#dfn-same-functionality) within a [set of Web pages](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html#dfn-set-of-web-pages) are identified consistently.

---

## H1 seem to have different styling depending on page?

<img width="441" alt="Screen Shot 2022-07-11 at 9 15 19 AM" src="https://user-images.githubusercontent.com/38330843/178276352-b06d72a0-aaf1-4813-bddc-9fe86bd66831.png">

<img width="404" alt="Screen Shot 2022-07-11 at 9 15 52 AM" src="https://user-images.githubusercontent.com/38330843/178276356-e882c156-b37e-4970-a374-a5b64ac26e3e.png">

## Options/links to select have different appearance depending where in site?

![Screen Shot 2022-07-11 at 9 37 18 AM](https://user-images.githubusercontent.com/38330843/178277092-7565029d-9d81-4013-a03a-d8d8e1665547.png)

![Screen Shot 2022-07-11 at 9 37 25 AM](https://user-images.githubusercontent.com/38330843/178277095-680e4d27-28c5-4ffa-86e0-d64578a27c7e.png)

![Screen Shot 2022-07-11 at 9 37 32 AM](https://user-images.githubusercontent.com/38330843/178277097-769a7afd-ba87-4cd4-8922-7885de7582af.png)

## Link styling varies?

![Screen Shot 2022-07-11 at 9 38 57 AM](https://user-images.githubusercontent.com/38330843/178277358-be79eadc-a286-4eea-b31a-d7e831a680fb.png)

![Screen Shot 2022-07-11 at 9 38 49 AM](https://user-images.githubusercontent.com/38330843/178277359-4c0d2e8f-5291-4583-99ad-aa85c2c338b9.png)

![Screen Shot 2022-07-11 at 9 38 43 AM](https://user-images.githubusercontent.com/38330843/178277363-1f4ba086-7867-47b2-b9a0-eba36f8f1873.png)

**Comment by anikbrazeau** [comment]
Linked to #180

**Comment by anikbrazeau** [comment]
Linked to #83

**Comment by dylanzheng94** [comment]
@anikbrazeau @emanelfy 
- if there are inconsistencies between H1's they will be fixed

- In terms of the link styling:
I could be wrong or be misinterpreting WCAG but I think the intent of this criterion is primarily to ensure varying text labels aren't used to describe the same function e.g. "Print" vs "Print page" 

The reason why there are different visuals for links in our website:
- Link with arrow (topics & subtopics)
- Link with border (collections)
- Link with text only (resource)

is in hopes that for users frequenting the website (and perhaps even new users) that the differentiation in styling communicates that the underlying resource behind those links are different. It breaks up the overwhelming sea of having only blue underlined text links everywhere. 

Now if to meet this criteria all links must visually look the same then we can have some discussions around that

**Comment by anikbrazeau** [comment]
@dylanzheng94 I don't think this would be a high priority issue.

I wonder if we’d want links to have consistency in whether they are underlined or whether they are a certain shade of blue etc.? Maybe it's just me, but there seem to be many different stylings for links? Are these intentional design choices / differences? 

I know they are slightly different in purpose, but to the user they might all just be links? I wonder if documenting why some are underlined and some are not, or why the approach changes, might help clarify this issue?

- **Body:** links are dark blue / underlined
- **Footer:** links are black / not underlined
- **Topics:** links are bright blue / not underlined
- **Resources:** links are bright blue / underlined
- **Collections:** links are bright blue / not underlined 
- **Sidebar:** links are dark blue / then bright blue on hover

**Comment by dylanzheng94** [comment]
@anikbrazeau 
Current rationale:
- Body links: default link style set by browser
- Footer links: to have the footer be differentiated visually from the rest of the page (more muted)
- Topics: not underlined because of the arrow icon 
- Resources: are underlined because there is no other visual indication of interaction other than colour
- Collections: not underlined because of the existing box visual and hover state
- Sidebar: same shade of blue as the other links above? then brighter blue on hover?

**Comment by daine** [comment]
Grooming notes:
- Clarify criteria for this
- Could be as simple as a 3 but without a definite acceptance criteria it could be 13+
